usr/sbin/registercloudguest

```diff
@@ -417,6 +419,7 @@ while not base_registered:
         stderr=subprocess.PIPE)
     res = p.communicate()
     if p.returncode != 0:
+        print('Registration failed, see /var/log/guestregister for details')
```

Comment:
We do not want to print the failure message for every update server we are trying, one summary message, you'll have to set a flag.